cpp/net/connection_pool.h

```diff
@@ -16,27 +16,40 @@ namespace cert_trans {
 namespace internal {
 
 
-struct evhttp_connection_deleter {
-  void operator()(evhttp_connection* conn) const {
-    evhttp_connection_free(conn);
+struct evhtp_connection_deleter {
+  void operator()(evhtp_connection_t* con) const {
+    evhtp_connection_free(con);
   }
 };
 
 
-typedef std::unique_ptr<evhttp_connection, evhttp_connection_deleter>
-    evhttp_connection_unique_ptr;
+typedef std::unique_ptr<evhtp_connection_t, evhtp_connection_deleter>
```

Comment:
Since this is no longer re-used, specifying it just once inline would be fine? Also, how about doing without `unique_ptr` altogether (for this), just make it a const raw pointer in `Connection`, and free it in `~Connection`?